packages/core/lib/serialise/ahoXml/serialiseToXml.ts

```diff
@@ -495,6 +498,14 @@ const xmlnsTags = {
   "@_xmlns:xsi": "http://www.w3.org/2001/XMLSchema-instance"
 }
 
+const mapPncExceptionsToXml = (exceptions: Exception[]): Br7PncErrorMessageString[] | undefined => {
+  const pncExceptions = exceptions.filter((exception) => "message" in exception) as PncException[]
+
+  return pncExceptions.length > 0
```

Comment:
This would read more clearly as an if/return than it does as ternary

```typescript
if(x) {
  return y
}
return
```

Reply:
We've included this changes in PR #1114, thanks!